### PR TITLE
Table changes to support OCD-2084

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: OCD-2084.sql [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+(
+  	id bigserial NOT NULL,
+  	age_count bigint NOT NULL,
+	test_participant_age_id bigint NOT NULL,
+  	creation_date timestamp without time zone NOT NULL DEFAULT now(),
+  	last_modified_date timestamp without time zone NOT NULL DEFAULT now(),
+  	last_modified_user bigint NOT NULL,
+  	deleted boolean NOT NULL DEFAULT false,
+  CONSTRAINT participant_age_statistics_pk PRIMARY KEY (id)
+);
```
Comment (andlar) [inline OCD-2084.sql]: Should there be a foreign key relationship to the test_participant_age table?

File: OCD-2084.sql [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+CREATE TABLE openchpl.participant_education_statistics
+(
+  	id bigserial NOT NULL,
+  	education_count bigint NOT NULL,
+	education_type_id bigint NOT NULL,
+  	creation_date timestamp without time zone NOT NULL DEFAULT now(),
+  	last_modified_date timestamp without time zone NOT NULL DEFAULT now(),
+  	last_modified_user bigint NOT NULL,
+  	deleted boolean NOT NULL DEFAULT false,
+  CONSTRAINT participant_education_statistics_pk PRIMARY KEY (id)
```
Comment (andlar) [inline OCD-2084.sql]: Foreign key to education_type?

File: OCD-2084.sql [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+CREATE TRIGGER participant_education_statistics_audit AFTER INSERT OR UPDATE OR DELETE on openchpl.participant_education_statistics FOR EACH ROW EXECUTE PROCEDURE audit.if_modified_func();
+CREATE TRIGGER participant_education_statistics_timestamp BEFORE UPDATE on openchpl.participant_education_statistics FOR EACH ROW EXECUTE PROCEDURE openchpl.update_last_modified_date_column();
+
+
+DROP TABLE IF EXISTS openchpl.participant_experience_statistics;
+
+CREATE TABLE openchpl.participant_experience_statistics
+(
+  	id bigserial NOT NULL,
+	experience_type_id bigint NOT NULL,
```
Comment (andlar) [inline OCD-2084.sql]: We can't foreign key this, right? Because it maps to three different types?

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
 # Release Notes
 
+## Version TBD
+_Date TBD_
+
+### Table Modifications
+* Add participant_gender_statistics table
+* Add participant_age_statistics table
+* Add participant_education_statistics table
+* Add participant_experience_statistics table
```
Comment (andlar) [inline RELEASE_NOTES.md]: Usually try to write these as things that are helpful for an end user, or at least have a top-line description that is. Maybe something like:
Added tables to support statistics related to SED Participant demographics:
* Gender
* Age
* Education Type
* Experience Levels

File: OCD-2084.sql (hunk 1 above)
Comment (tmy1313) [reply OCD-2084.sql]: I'll make this change.  I waffled back and forth about this.  Don't remember my logic for not adding it...

File: OCD-2084.sql (hunk 2 above)
Comment (tmy1313) [reply OCD-2084.sql]: I'll make this change.  I waffled back and forth about this.  Don't remember my logic for not adding it...

File: OCD-2084.sql (hunk 3 above)
Comment (tmy1313) [reply OCD-2084.sql]: Right - we could create a new table with the 3 types if we wanted to.

File: RELEASE_NOTES.md (hunk 4 above)
Comment (tmy1313) [reply RELEASE_NOTES.md]: I'll make this update.

**Pull request merged**